src/plotman/analyzer.py

```diff
@@ -89,14 +90,27 @@ def analyze(logfilenames, clipterminals, bytmp, bybitfield):
                     if clipterminals and is_first_last:
                         pass  # Drop this data; omit from statistics.
                     else:
-                        data.setdefault(sl, {}).setdefault('total time', []).append(float(m.group(1)))
-                        for phase in ['1', '2', '3', '4']:
-                            data.setdefault(sl, {}).setdefault('phase ' + phase, []).append(phase_time[phase])
-                        data.setdefault(sl, {}).setdefault('%usort', []).append(100 * n_uniform // n_sorts)
+                        total_time = float(m.group(1))
+
+                # Copy Time.  Sample log line:
+                # Copy time = 958.194 seconds. CPU (6.290%) Sat May 22 17:39:57 2021
+                m = re.search(r'^Copy time = (\d+.\d+) seconds.*', line)
+                if m:
+                    copy_time = float(m.group(1))
+
+                # Rename plot.  Sample log line:
+                # Renamed final file from "/mnt/d1/a4.plot.2.tmp" to "/mnt/d2/a4.plot"
+                m = re.search(r'^Renamed final file.*', line)
```

Comment:
Is this present regardless of what combinations of tmp/tmp2/dst are specified?

Reply:
Yes, and if tmp same as dst it will show 0 sec because of simple renaming.